_dashboards/management/data-sources.md

```diff
@@ -0,0 +1,73 @@
+
+## Create a data source connection
+
+A data source connection specifies the parameters needed to connect to a data source. These parameters form a connection string for the data source. Using Dashboards, you can add new data source connections or manage existing ones.
+
+The following steps guide you through the basics of creating a data source connection:
+
+1. From the OpenSearch Dashboards main menu, select **Dashboards Management** > **Data sources** > **Create data source connection**. The user interface is shown in the following image.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/data-source-UI.png" alt="Connecting a data source UI" width="700"/>
+
+2. Add information to each field to configure **Connection Details** and **Authentication Method**. 
+   
+   - Under **Connection Details**, enter a title and endpoint URL. For this tutorial, use the URL `http://localhost:5601/app/management/opensearch-dashboards/dataSources`. Entering a description is optional.
+
+   - Under **Authentication Method**, select an authentication method from the dropdown list. Once an authentication method is selected, the applicable fields for that method appear. You can then enter the required details. The authentication method options are:
+       - **No authentication**: No authentication is used to connect to the data source.
+       - **Username & Password**: A basic username and password are used to connect to the data source.
+       - **AWS SigV4**: An AWS Signature Version 4 authenticating request is used to connect to the data source. AWS Signature Version 4 requires an access key and a secret key. 
+         - For AWS Signature Version 4 authentication, first specify the **Region**. Next, select the OpenSearch service in the **Service Name** list. The options are **Amazon OpenSearch Service** and **Amazon OpenSearch Serverless**. Last, enter the **Access Key** and **Secret Key** for authorization.
+
+    After you have populated the required fields, the **Test connection** and **Create data source** buttons become active. You can select **Test connection** to confirm that the connection is valid.
+
+3. Select **Create data source** to save your settings. The connection is created. The active window returns to the **Data Sources** main page, and the new connection appears in the list of data sources.
+
+4. To delete a data source connection, select the check box to the left of the data source **Title** and then select the **Delete 1 connection** button. Selecting multiple check boxes for multiple connections is supported. An example of the user interface is shown in the following image.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/delete-data-source.png" alt="Deleting a data source UI" width="700"/>
+
+### Modify a data source connection
+
+To make changes to the data source connection, select a connection in the list on the **Data Sources** main page. The **Connection Details** window opens.
```

Comment:
Confirm the capitalization of the UI elements here and in the following paragraph.

Reply:
Confirmed. The UI uses both Title Case and Sentence case in the UI. Per UX, the Engineering team handles making the UI changes once they've added the UI to their dev environment. We need to resolve this issue because this isn't a new feature. UI copy edits were provided to Engineering and I think those changes weren't made.

Reply:
@kgcreative For your attention.

Reply:
@vagimeli -- can you open an issue in the appropriate repo for text changes? We should be able to update that for 2.12 -- Alternatively, If you send me annotated screenshots I should be able to do that.